### PR TITLE
update pull_request.yml to decrease command to 4g

### PR DESCRIPTION
**Pull request by Ginxo**
related to https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/1421

### CODE REVIEW

**Comment by mareknovotny** [comment]
shouldn't we actually decrease the memory to 4G?

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       with:
         parent-dependencies: 'kie-wb-playground'
         child-dependencies: 'drools-wb'
-        build-command: 'mvn -e -nsu -Dfull clean install -Prun-code-coverage -Pwildfly -Dcontainer=wildfly -Dcontainer.profile=wildfly -Dintegration-tests=true -Dmaven.test.failure.ignore=true -Dwebdriver.firefox.bin=/opt/tools/firefox-60esr/firefox-bin -DjvmArgs="-Xms1g -Xmx5g"'
+        build-command: 'mvn -e -nsu -Dfull clean install -Prun-code-coverage -Pwildfly -Dcontainer=wildfly -Dcontainer.profile=wildfly -Dintegration-tests=true -Dmaven.test.failure.ignore=true -Dwebdriver.firefox.bin=/opt/tools/firefox-60esr/firefox-bin -DjvmArgs="-Xms1g -Xmx11g"'
```
Comment (mareknovotny) [inline .github/workflows/pull_request.yml:15]: better to use 4G as on Jenkins

**Comment by mareknovotny** [comment]
ok, it failed again on the same error.

I looked at the OS resources, which are on Github Actions by default and it is 7G for RAM, 14GB for disk space
https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners

**Comment by domhanak** [comment]
@Ginxo Closing as out of date. Please reopen if needed.

**Pull request closed without merging**